Tests/SwiftGRPCTests/EchoTests.swift

```diff
@@ -38,7 +38,11 @@ class EchoTests: BasicEchoTestCase {
 }
 
 class EchoTestsSecure: EchoTests {
-  override var secure: Bool { return true }
+  override var security: Security { return .ssl }
+}
+
```

Comment:
Please add this test class to LinuxMain.swift.

Reply:
LinuxMain.swift is another generated file. Am I in for a world of hurt?

Reply:
That file shouldn't be autogenerated?

Reply:
Oh you're right. I was looking at LinuxMain.swift in swift-protobuf. 😌